src/shared/components/search.tsx

```diff
@@ -470,23 +475,29 @@ export class Search extends Component<any, SearchState> {
     }
 
     // Push the search results
-    combined.push(
-      ...this.state.searchResponse?.comments?.map(e =>
-        this.commentViewToCombined(e)
-      )
+    pushNotNull(
+      combined,
+      this.state.searchResponse?.comments
+        ?.filter(e => e)
```

Comment:
There were lint errors on lines like `comments.push(...this.state.searchResponse?.comments)`, saying "Unsafe usage of optional chaining. If it short-circuits with 'undefined' the evaluation will throw TypeError  no-unsafe-optional-chaining". So i added this helper method, to avoid littering the whole code with null/undefined checks.

Reply:
I mean the filter line specifically, because it does nothing. I'll try to pull it down and see if I get the same issue when using `yarn lint`

Reply:
Okay yes I just tried removing these filter lines, and yarn lint still passes fine.

Reply:
oh thats true. i think i added those as a null check, before using pushNotNull()